base/hugegraph/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM hugegraph/hugegraph:1.0.0
```

Comment:
Please create a new folder for this image like <https://github.com/vulhub/vulhub/tree/master/base/activemq>.